suite-native/module-device-settings/src/components/UnpairBluetoothDeviceCard.tsx

```diff
@@ -0,0 +1,31 @@
+import { useNavigation } from '@react-navigation/native';
+
+import { CompactCardWithIconLayout } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import {
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+
+type NavigationProp = StackNavigationProps<
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes.DeviceSettings
+>;
+
+export const UnpairBluetoothDeviceCard = () => {
+    const navigation = useNavigation<NavigationProp>();
+
+    const handleRedirectToUnpairScreen = () => {
```

Comment:
The name is misleading from my point of view. This action does not handle redirection, it navigates:
```suggestion
    const navigateToUnpairScreen = () => {
```

Reply:
Fixed